src/js/profile/category.js

```diff
@@ -0,0 +1,43 @@
+import {Subcategory} from "./subcategory";
+
+let categoryTemplate = null;
```

Comment:
Why are these global and not part of the class?

Reply:
previously we were removing the elements from dom in the parent class, so before removing those object from the dom I had to get those clone objects in the parent class too. now with the new design we have a hidden styles class so we don't need to remove elements from dom anymore. I moved those global objects into their classes